### DISCUSSION TITLE
ExportAirr v_sequence_alignment

### DISCUSSION THREAD

**Discussion by gabrielathenes** [milaboratory/mixcr#1605] [Q&A] [answered]
Hello,

Is there a way to have the 'v_sequence_alignment' and 'j_sequence_alignment' directly using the alignment file .vdjca and the ExportAirr function ? 

Additionally, I would like to get the nucleotides of "v_sequence_alignment" and "v_germline_alignment" that belong to the cdr3, any idea on how I could do this ? 

Thanks a lot !

**Comment by mizraelson** [comment, marked as answer]
The mixcr `exportAIRR` function can take a `.vdjca` file as input. In the AIRR format, there is no option to export the CDR3 part of the V gene. However, this can easily be done with the standard MiXCR export functions, `exportAlignments` for `.vdjca` files or `exportClones` for `.clns`/`.clna` files:

`-nFeature {FR3End:VEnd} germline` for the germline CDR3 part of the V gene
`-nFeature {FR3End:VEndTrimmed}` for the V part of the CDR3 in the actual clone

Example:

```
mixcr exportAlignments \
-nFeature {FR3End:VEnd} germline \
-nFeature {FR3End:VEndTrimmed} \
input.vdjca \
alignments.tsv
```

Note! It is important to use the latest [develop version](https://link.milaboratory.com/software/mixcr/mixcr-develop.zip) for this, as it contains an important fix.
